### DISCUSSION TITLE
Welcome to fvwm3 Discussions!

### DISCUSSION THREAD

**Discussion by ThomasAdam** [fvwmorg/fvwm3#335] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by danespen** [comment]
Not sure what this is about, but, here are the headers on your email:

To: Thomas Adam <notifications@github.com>
Cc: fvwmorg/fvwm3 <fvwm3@noreply.github.com>,  fvwmorg/fvwm3 <reply@reply.github.com>,  Subscribed <subscribed@noreply.github.com>

I assume the "To" will work and I guess but I'm not sure who it goes to.
It looks like it goes to you.
In the "Cc" line, I see 2 NOREPLY addresses.

What's wrong with fvwm@fvwm.org?


Thomas Adam <notifications@github.com> writes:

> 👋 Welcome!
>
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
>
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>  build together 💪.
>
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
>

-- 
Dan Espen

**Reply by ThomasAdam** [reply]
Hi @danespen 

I'm not sure about the headers -- I  mean, they're not something I can control or set.  Will have to check if that's going to be a problem.

Thanks for letting me know.
